create_docker_sysext.sh

```diff
@@ -3,25 +3,37 @@ set -euo pipefail
 
 ARCH="${ARCH-x86_64}"
 OS="${OS-flatcar}"
+ONLY_CONTAINERD="${ONLY_CONTAINERD-0}"
+ONLY_DOCKER="${ONLY_DOCKER-0}"
+FORMAT="${FORMAT-squashfs}"
```

Comment:
How about being stricter and validating this variable to be either "squashfs" or "ext4" or "btrfs" early?

Reply:
:+1: